contentful_management/webhook_resource_proxy.py

```diff
@@ -0,0 +1,50 @@
+"""
+contentful.webhook_resource_proxy
+~~~~~~~~~~~~~~~~~~~~~~~
+
+This module implements the WebhookResourceProxy class.
+
+:copyright: (c) 2017 by Contentful GmbH.
+:license: MIT, see LICENSE for more details.
+"""
+
+
+class WebhookResourceProxy(object):
+    """Base Class for Webhook related Resource Proxies"""
```

Comment:
It's the same everywhere, so for the sake of keeping it as it already is, I'll leave it here, maybe on a cleanup commit later we can change all the docs to have different capitalization.